### ISSUE TITLE
URL https://us-rse.org/find-an-rse/ in description 404s

### ISSUE THREAD

**Issue by yarikoptic** [USRSE/find-an-rse#2]

**Comment by vsoch** [comment]
Good catch! This was originally at usrse, and intended for community- repositories that I wanted to create for each community. The project wasn't really supported by the org so I moved it here for safekeeping, and haven't thought about how to best go about updating it. The correct URL would be https://vsoch.github.io/find-an-rse/, but since the community links also are different, I doubt the search will work.

So here are my thoughts - this would probably do best to move to the rseng organization, but I'd want to have a sense of where to take the project first. Since we have the usrse-map https://us-rse.org/usrse-map/ that would arguably be a high level way to "Find an RSE" but without any personal information. I think it's hugely unlikely that people would want to reveal personal information, or volunteer their own information in any respect actually, but what about some way to (also using the usrse metadata) have a way to connect people? What could we do with the membership form data that 1) wouldn't be violating anyone's privacy but 2) if I came to an interface like this, I could do a search for some kind of RSE / domain I'm interested in, and then (discreetly?) get those people contacted?

**Comment by yarikoptic** [comment]
re map: what the size of the blob represents? brown has 6 but much smaller than the largest blob with 1 ;-)

**Comment by yarikoptic** [comment]
re questions: I guess RSE registration/submission should have a question? or if it is all discoverable/links to github then it should just be made available to github-authenticated ppl?

**Comment by vsoch** [comment]
The largest blob (Stanford) has 21 - which blob has 1 that is larger than Brown?

**Comment by vsoch** [comment]
The RSE registration information is totally private - the CI jobs grab the high level institutions by way of a separate sheet that just exposes that column of information.

**Comment by yarikoptic** [comment]
> The largest blob (Stanford) has 21 - which blob has 1 that is larger than Brown?

I was blind and somehow saw "1", please disregard me ;)

**Comment by vsoch** [comment]
Haha, no worries. :)
